weblate/accounts/views.py

```diff
@@ -441,6 +441,20 @@ def get_initial_contact(request):
     return initial
 
 
+@csrf_exempt
+@never_cache
+def saml_metadata(request):
+    complete_url = reverse('social:complete', args=("saml",))
+    saml_backend = social_django.utils.load_backend(
+        social_django.utils.load_strategy(request),
+        "saml", complete_url)
+    metadata, errors = saml_backend.generate_metadata_xml()
+    if not errors:
+        return HttpResponse(content=metadata,
+                            content_type='text/xml')
+    return HttpResponseServerError(content=', '.join(errors))
```

Comment:
Can you please add some basic test for this? At least that it returns well formed XML.

Reply:
Also here is missing import:

```
./weblate/accounts/views.py:455:12: F821 undefined name 'HttpResponseServerError'
```